src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpMultilineCheck.java

```diff
@@ -92,14 +125,70 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * Example of violation from the above config:
+ * Example:
```

Comment:
this example is very weird, as almost no violations at all, clear signal that regexp is damaged.

please update config to use `.*?`

----

please move paragraph `Note: Beware of the greedy regular expression used in the above` to be below this example.

Reply:
Yeah, right

Reply:
You mean i have to update regexp to "System\.out.*?print\(" ?

Reply:
yes, in other case example to show user config that do bad job, but examples should be ready to copy paste for user to use.